package.json

```diff
@@ -50,13 +50,17 @@
     "start:server": "webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
     "watch:templates": "gulp watch",
     "start": "npm run build:templates && run-p start:server watch:templates",
-    "test": "karma start karma.config.js",
+    "lint": "eslint .",
+    "test:unit": "karma start karma.config.js",
     "build": "npm run build:templates && webpack --mode production --config config/webpack.prod.mjs",
     "start:prod-local": "cd dist && http-server -p 4200 -c-1 -b",
     "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
-    "lint": "eslint .",
+    "test:e2e": "playwright test",
+    "test:e2e:debug": "playwright test --project chromium --headed",
+    "test:e2e:report": "npx playwright show-report",
+    "test": "run-s lint test:unit test:e2e",
```

Comment:
As discussed in person, if unit tests do not pass (for example, because there aren't any atm), we should remove `test:unit` from the `test` script (leaving only `lint` and `test:e2e`) and then work on a follow-up PR to add some unit tests and fix our setup (and add `test:unit` back into `test`).

Reply:
Fixed